<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vidStreamer</class>
 <widget class="QWidget" name="vidStreamer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>vidStreamer</string>
  </property>
  <widget class="QPushButton" name="pbVideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPause">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>290</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>#SpiNNaker chips:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSpiNN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>226</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SpiNN3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SpiNN5</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbConfigure">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>226</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbNchips">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>186</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>48</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Operator Type</string>
    </property>
    <widget class="QRadioButton" name="rbSobel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sobel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbLaplace">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Laplace</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Filtering Option</string>
    </property>
    <widget class="QRadioButton" name="rbFilterOff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbFilterOn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Sharpening Option</string>
    </property>
    <widget class="QRadioButton" name="rbSharpOff">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSharpOn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbImage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="cbTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Workers ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Network Configuration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Block Info</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Workload</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frame Info</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Performance Measurement</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Test:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbSendImage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Image</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
